src/videoCache.js

```diff
@@ -162,6 +175,20 @@ export function getCacheUrl(id) {
   return `${config.getConfig('cache.url')}?uuid=${id}`;
 }
 
+export const storeLocally = (bid) => {
+  const vastValue = getVastXml(bid);
+  const strategy = config.getConfig('cache.strategy') || BLOB;
+  const bidVastUrl = {
+    [DATA]: (xml) => 'data:text/xml;base64,' + btoa(xml),
+    [BLOB]: (xml) => URL.createObjectURL(new Blob([xml], { type: 'text/xml' }))
+  }[strategy](vastValue);
+
+  bid.vastUrl = bidVastUrl;
+  bid.videoCacheKey = generateUUID();
```

Comment:
this should be shared with the "normal" cache flow - so that they don't get out of sync if the logic changes